libraries/engage/components/MessageBar/MessageBar.jsx

```diff
@@ -41,12 +43,14 @@ const MessageBar = ({ messages, classNames }: Props) => (
         <div
           key={`${type}-${message}`}
           className={css(
-            classNames.message,
             styles[type],
+            classNames.message,
             Icon ? styles.withIcon : null
           )}
         >
-          {Icon && <Icon className={css(classNames.icon, styles.icon).toString()} /> }
+          {Icon && (
+            <Icon className={css(classNames.icon, styles.icon).toString()} />
```

Comment:
The designs show an info icon for every message. It seems that it was already implemented for error messages within the StoreListSearch, but not here.

Reply:
We leave that out, as discussed.